spec/models/legislator_spec.rb

```diff
@@ -61,6 +61,93 @@
     end
   end
 
+  describe ".recheck_reps_with_us" do
+    before do
+      FactoryGirl.create(:representative, bioguide_id: "D000563", with_us: false)
+      FactoryGirl.create(:representative, bioguide_id: "foo", with_us: false)
+      FactoryGirl.create(:representative, bioguide_id: "bar", with_us: true)
+      Legislator.recheck_reps_with_us
+    end
+
+    it "sets with_us property for all reps" do
+      expect(Legislator.find_by(bioguide_id: "D000563").with_us).to be true
+      expect(Legislator.find_by(bioguide_id: "foo").with_us).to be false
+      expect(Legislator.find_by(bioguide_id: "bar").with_us).to be false
+    end
+  end
+
+  describe "#update_reform_status" do
```

Comment:
@Rio517 these next three tests are basically duplicates (this functionality is tested in reps_with_us_spec). Do you think I should remove them?